### ISSUE TITLE
SOURCE CODE FOR IMAGE MAP USED IN NEWSPAPER

### ISSUE THREAD

**Issue by nitzz** [simplegeo/polymaps#137]
I want source code for image viewer as map which on click on image areas...show same image as of are selected.For more please see link
http://epaper.jagran.com/homepage.aspx

I want to create same thing as in the link below.

Thanks,
Nitzz

**Comment by eambajogai** [comment]
Yes I trying many month searching source or code for newspaper websites but i have not got this thing anyone help to me hr.shankarchavan@gmail.com or 9921042422 like http://epaper.jagran.com/homepage.aspx
